tip-wlan/charts/wlan-ssc-service/templates/configmap.yaml

```diff
@@ -11,8 +11,10 @@ data:
             contact-points = [ "{{ include "cassandra.service" . }}:9042" ]
             load-balancing-policy.local-datacenter = datacenter1
             session-keyspace = tip_wlan_keyspace
+            request.consistency = LOCAL_QUORUM
```

Comment:
Why is this change required?

Reply:
For a deployment with a single replica Cassandra, this setting doesn't matter. But if you were to deploy 3 replicas of Cassandra, then using "LOCAL_QUORUM" will allow ssc to continue to function with the other 2 available Cassandras when cassandra-0 goes down.